libs/constants/index.js

```diff
@@ -2,7 +2,6 @@ const VALID_LOCALES = new Map(
   [
     "ar",
     "bg",
-    "bm",
```

Comment:
There are 10 documents in MySQL that has this locale:
```
mysql> select locale, slug, is_redirect from wiki_document where locale = 'bm';
+--------+-----------------------------------------------------------+-------------+
| locale | slug                                                      | is_redirect |
+--------+-----------------------------------------------------------+-------------+
| bm     | Mozilla                                                   |           0 |
| bm     | Web                                                       |           0 |
| bm     | Web/JavaScript                                            |           0 |
| bm     | Web/JavaScript/Reference                                  |           0 |
| bm     | Web/JavaScript/Guide                                      |           0 |
| bm     | Web/JavaScript/Guide/Grammar_and_types                    |           0 |
| bm     | Web/JavaScript/Reference/Deprecated_and_obsolete_features |           0 |
| bm     | Web/JavaScript/New_in_JavaScript                          |           0 |
| bm     | Web/JavaScript/New_in_JavaScript/1.1                      |           0 |
| bm     | Web/JavaScript/Guide/Details_of_the_Object_Model          |           0 |
+--------+-----------------------------------------------------------+-------------+
10 rows in set (0.00 sec)
```
but when you run the importer it's all ignored and never dumped. Can't remember why! Let's find out.

Reply:
Silly me! They're all deleted:
```
mysql> select locale, slug, is_redirect, deleted from wiki_document where locale = 'bm';
+--------+-----------------------------------------------------------+-------------+---------+
| locale | slug                                                      | is_redirect | deleted |
+--------+-----------------------------------------------------------+-------------+---------+
| bm     | Mozilla                                                   |           0 |       1 |
| bm     | Web                                                       |           0 |       1 |
| bm     | Web/JavaScript                                            |           0 |       1 |
| bm     | Web/JavaScript/Reference                                  |           0 |       1 |
| bm     | Web/JavaScript/Guide                                      |           0 |       1 |
| bm     | Web/JavaScript/Guide/Grammar_and_types                    |           0 |       1 |
| bm     | Web/JavaScript/Reference/Deprecated_and_obsolete_features |           0 |       1 |
| bm     | Web/JavaScript/New_in_JavaScript                          |           0 |       1 |
| bm     | Web/JavaScript/New_in_JavaScript/1.1                      |           0 |       1 |
| bm     | Web/JavaScript/Guide/Details_of_the_Object_Model          |           0 |       1 |
+--------+-----------------------------------------------------------+-------------+---------+
10 rows in set (0.00 sec)
```